api/Insight_API_documentation.md

```diff
@@ -0,0 +1,1200 @@
+            "valueIn": 4248.48951622,
+            "fees": 0.010165
+        },
+
+    ]
+}
+```
+
+<br/>
+
+### /tx/send/ (POST)
+
+**URL:**  ```POST /tx/send {"rawtx":[RAWTXDATA]"}``` **REVIEW**
+
+**Description:** Broadcasts transaction to network (**REVIEW**).
+
+**Parameters:**
+
+| Parameter           | Type                   |  Description                   | 
+| -------------------- | ---------------------- | ---------------------- | 
+| rawtx              | `string`      |   `Signed transaction as hex string`       |  
+
+
+**Request Example:**
+
+```
+curl -X POST \
+ https://alpha.dcrdata.org/insight/api/tx/send \
+ -H 'Cache-Control: no-cache' \
+ -H 'Content-Type: application/json' \
+ -H 'Postman-Token: bf2e16a1-b6d2-4b2f-b6e5-9ea7bd3df5b1' \
+ -d '{"rawtx":"010000000184de71690b97b4cbac6e723570a25a56295b30aa26345bfc40c0609c87e23f1d0100000000ffffffff02809698000000000000001976a9143eb656115197956125365348c542e37b6d3d259988ac00811b2c0000000000001976a914e4c9ada86ba67b2c082fac1aa09fb17fccc6833688ac000000000000000001ffffffffffffffff00000000ffffffff6b483045022100b5020004e60a4d26c99a00316ed4f51131b99838fd8fab936d55f9475719bb3b022063b86596adba004367a138f62c73d59f8f3b2cded4b10b5fc0ebe92a4cf41c840121029ec6a82a9646c090decf20806029f332cc11a5c6fa17c943ee72c31b9707a433"}'
+```
+**Request Response:**
+```
+{
+    "rawtx": "18a4eeed058c2266512863d03f79651cd38d94d6d682ae3f1e4aad0178c6998f"
+}
+```
+<br/>
+
+## Addresses 
+
+Methods that work with addresses.
+
+### /addr/
+
+**URL:**  ```GET /addr/{address}?noTxList=1&from=N&to=M```
+
+**Description:** Retrieves addresses in range `N` to `M`**REVIEW WORDING** 
+```NOTE: from @pappacarp comment, suggested format: [:addr][?noTxList=1][&from=&to=]```
+
+**Parameters:**
+
+| Parameter           | Type                   |  Description                   | 
+| -------------------- | ---------------------- | ---------------------- | 
+| hash              | `string`      |   Address    |  
+| N              | `int64`      |   (optional) Address start **REVIEW/SUGGEST ALTERNATIVE**       |  
+| M              | `int64`      |   (optional) Address end **REVIEW/SUGGEST ALTERNATIVE**       |  
+| noTxList              | `boolean`      |   (optional) If `noTxList` = '0', response includes a `noTxList`. If `noTxList` = '1', response will not include a `noTxList`      |  
+
+**Request Example:**
+
+```GET /addr/DshRLMBrqyDmMpp9gcz3bLgCnjMa75mrHsS?from=100&to=200?noTxList=1 ```
+
+**Request Response:**
+
+```
+{
+    "addrStr": "DshRLMBrqyDmMpp9gcz3bLgCnjMa75mrHsS",
+    "balance": 0,
+    "balanceSat": 0,
+    "totalReceived": 93.00360718,
+    "totalReceivedSat": 9300360718,
+    "totalSent": 93.00360718,
+    "totalSentSat": 9300360718,
+    "unconfirmedBalance": 0,
+    "unconfirmedBalanceSat": 0,
+    "unconfirmedTxApperances": 0,
+    "txApperances": 4
+}
+```
+
+<br/>
+
+### /addr/ (balance)
+
+**URL:**  ```GET /addr/{address}/balance```
+
+**Description:** Retrieves address balance.
+
```

Comment:
in atoms which is the smallest unit of DeCred

Reply:
Went with this. Just wanted to double check what an atom is. Didn't find the concept of an atom skimming the docs, and had to find out what an atom was via google->reddit.
`Retrieves address balance in atoms (the smallest unit of Decred; 1 DCR = 100,000,000 atoms).`

Reply:
Yes, 1 DCR is 1e8 atoms.  This is formalized in https://godoc.org/github.com/decred/dcrd/dcrutil#Amount

Reply:
@chappjc , this seems like a detail that should be added to the Decred docs. Before I submit an issue in dcrdocs though, is there some big wording debate I'm not aware of?

Reply:
No debate that I'm aware of.  Except that I briefly advocated to rename atoms to tacos as a node to TacoTime.  :)  Or maybe "decreds".

Reply:
If 1 DCR contained 100,000,000 tacos I'd be buying lol. 
Will submit PR to dcrdocs that adds atoms and see if anyone objects.